src/features/defi/providers/fox-farming/hooks/useFoxFarming.ts

```diff
@@ -148,7 +147,7 @@ export const useFoxFarming = (
     const data = encodeFunctionData({
       abi: uniV2LPContract.abi,
       functionName: 'approve',
-      args: [contractAddress, BigInt(MaxUint256.toString())],
+      args: [contractAddress, BigInt(maxUint256.toString())],
```

Comment:
assuming maxUint256 is already `BigInt`
```suggestion
      args: [contractAddress, maxUint256],
```

Reply:
big 🧠  fbb0f62ee7